plugins/database/influxdb/influxdb_test.go

```diff
@@ -0,0 +1,315 @@
+package influxdb
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/vault/builtin/logical/database/dbplugin"
+
+	"fmt"
+
+	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/ory/dockertest"
```

Comment:
Would you be willing to organize these imports next time you're making a pass at this code? Not 100% necessary so if it's a bother, don't worry about it!